cartoframes/carto_vl/layer.py

```diff
@@ -106,13 +107,13 @@ def _parse_interactivity(interactivity):
         return None
     elif isinstance(interactivity, dict):
         return {
-          'event': interactivity.get('event', event_default),
-          'header': interactivity.get('header', None),
-          'values': interactivity.get('values', None)
+            'event': interactivity.get('event', event_default),
+            'header': interactivity.get('header', None),
```

Comment:
Probably my fault, but the `None` is implied if there's no second argument in the `.get`, so it's not needed here

Reply:
The only change in this file is regarding PEP 8

Reply:
No worries